python/pylibcudf/pylibcudf/tests/test_nvtext_stemmer.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+from utils import assert_column_eq
+
+
+@pytest.fixture(scope="module")
+def input_col():
+    arr = ["trouble", "toy", "sygyzy"]
```

Comment:
Not a bug, really, but the word is syzygy. Maybe we can fix this in C++ tests too.

![09716570b4661b35a8e2a8be336d21ee](https://github.com/user-attachments/assets/5647e562-b0cf-4728-ba5d-c66109e5e461)

```suggestion
    arr = ["trouble", "toy", "syzygy"]
```

Reply:
Ha! TIL